update.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+set -e -u
+
+MB_RNGPY_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+cd "$MB_RNGPY_ROOT"
+
+usage="Usage: $0 [<python command>]"
+
+if [ $# -gt 1 ]
+then
+  echo Error: Too many arguments
+  echo $usage
+  exit -1
+fi
+
+if [ $# -eq 1 ]
+then
+  python_command=$1
+  if ! type $python_command &>/dev/null
+  then
+    echo Error: $python_command command not found
+    echo $usage
+    exit -1
+  fi
+else
+  if type python2 &>/dev/null
+  then
+    python_command=python2
+  elif type python &>/dev/null
+  then
+    python_command=python
+  else
+    echo Error: python command not found
+    echo $usage
+    exit -1
+  fi
+fi
+
+if [[ `$python_command --version` =~ "Python 2." ]]
+then
+  echo Error: $python_command is not Python 2.x
+  echo $usage
+  exit -1
+fi
+
+if ! type trang &>/dev/null
+then
+  echo Error: trang command not found
+  echo Please install trang package
+  exit -1
+fi
+
+if ! type twine &>/dev/null
+then
+  echo Error: twine command not found
+  echo Please install twine package
+  exit -1
+fi
+
+################################################################################
+echo Setting virtual environment up
+
+virtualenv -p $python_command venv
+set +u
+source venv/bin/activate
+set -u
+pip install -r requirements.txt
```

Comment:
I'm not sure what MeB's policy on this is in general, but I think this would also be a good time to be explicit about the version numbers of the dependencies in requirements.txt. The diff in #3 contained a bit of noise (indentation, line breaks, ..) that I believe was caused simply by using a different version of generateDS than before. Since the virtual environment is not removed and `pip install ...` doesn't upgrade already installed packages unless explicitly told to do so via `-U`, this could pick up an environment with older versions of the dependencies than a new virtual environment would pull in. This is not a big deal, but something to be aware of.

Reply:
Added `-U` which might not advocate for less noise though...
https://github.com/metabrainz/mb-rngpy/blob/4a718d6c036e708d62bce666b04eece0d901d2f3/update.sh#L69

Will review dependency versions later on.

Reply:
Given that `generateDS` and `lxml` are quite actively developed and have frequent bugfixing releases, that these are used once in a while, that used versions are written in the header of generated files, and that no manual step is involved, I think it is worth following latest releases even though it sometimes implies a bit of noise in the diff. If something goes wrong, it is still possible to manually regenerate files by following script steps with previous versions of dependencies.

Reply:
On second thoughts, I changed this with commit https://github.com/metabrainz/mb-rngpy/commit/607b948d3f98debb18c4c69b78df29108ea6b1a0 because I realized `lxml` version installed for packaging `mb-rngpy` and `lxml` version for running `mb-rngpy` have to be the same (or at the very least compatible). And since we require a fixed version for `lxml`, it is safer to require a fixed version for `generateDS` that is known to work with.